katello/hooks/pre/11-clear_pulpcore_data_feature.rb

```diff
@@ -0,0 +1,12 @@
+# See bottom of the script for the command that kicks off the script
+
+def clear_pulpcore_content
+  if File.directory?('/var/lib/pulp/docroot')
+    `rm -rf /var/lib/pulp/docroot/*`
```

Comment:
I don't like shelling out, especially since this discards any exit status. Perhaps `FileUtils.rm_rf` should be used? It also has docs on the security implications. Not sure if `rm -rf` has those too.

Reply:
@ekohl I've mirrored the `clear_pulp_content` method, should we address this issue there first?

And yeah the DB is cleared during the main reset task. Looks like clearing out the pulp content historically has been an add-on.